server.go

```diff
@@ -177,3 +181,18 @@ func (s *Server) DeserializeKE2(ke2 []byte) (*message.KE2, error) {
 func (s *Server) DeserializeKE3(ke3 []byte) (*message.KE3, error) {
 	return s.Parameters.DeserializeKE3(ke3)
 }
+
+// SetAKEState sets the interal state of the AKE server from the given bytes
+func (s *Server) SetAKEState(state []byte) error {
+	if len(state) != s.MAC.Size()+s.KDF.Size() {
+		return errors.New("invalid state length")
```

Comment:
Let's define that error up above like so

```
var (
        // ErrAkeInvalidClientMac indicates that the MAC contained in the KE3 message is not valid in the given session.
        ErrAkeInvalidClientMac = errors.New("failed to authenticate client: invalid client mac")

        // ErrInvalidState indicates that the given state is not valid due to a wrong length.
        ErrInvalidState = errors.New("invalid state length")
)
```

```suggestion
		return ErrInvalidState
```